test/integration/test_integration_basics.py

```diff
@@ -33,7 +33,7 @@ def test_h1(self):
                      1400 if IntegrationTestBase.ip_version == IpVersion.IPV6 else 1500)
     self.assertEqual(counters["upstream_rq_pending_total"], 1)
     self.assertEqual(counters["upstream_rq_total"], 25)
```

Comment:
Why not add a helper to verify that the map of metrics and values return are the same as an expected map.

Reply:
Done!